sts_backend/ml_logic/preprocessor.py

```diff
@@ -1,16 +1,35 @@
+
 import numpy as np
 import pandas as pd
-
+import os
+from tqdm import tqdm
 from colorama import Fore, Style
+import librosa
+
+# from sklearn.pipeline import make_pipeline
+# from sklearn.compose import ColumnTransformer, make_column_transformer
+# from sklearn.preprocessing import OneHotEncoder, FunctionTransformer
+
+def preprocess_features() -> np.ndarray:
+
+    mfcc_features = []
+    directory = os.path.join(os.pardir,'audio','splits')
+    # directory = os.path.join(os.pardir,os.pardir)
+    print(directory)
+    for clip in tqdm(os.listdir(directory)):
+        audio, sample_rate = librosa.load(os.path.join(directory, clip) , sr=8000)
+        mfccs = librosa.feature.mfcc(y=audio, sr=sample_rate, n_mfcc=20)
+        if mfccs.shape == (20,47):
+            mfcc_features.append(mfccs)
+        else:
+            print(f"{clip}:{mfccs.shape}")
+        print(clip)
+    x_mfcc = np.stack(mfcc_features)
 
-from sklearn.pipeline import make_pipeline
-from sklearn.compose import ColumnTransformer, make_column_transformer
-from sklearn.preprocessing import OneHotEncoder, FunctionTransformer
+    return x_mfcc
```

Comment:
```suggestion
    return np.stack(mfcc_features)
```
I think we can remove line 27 and make it return `np.stack(mfcc_features)` as well.